recipe/meta.yaml

```diff
@@ -1,39 +1,42 @@
-{% set version = "1.1.3" %}
+{% set name = "scikit-learn" %}
+{% set version = "1.2.0" %}
 
 package:
-  name: scikit-learn
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: b7c0a9fb8dfcefcfc7ef8fe02a064d194980251d57efaea972cb76005afb3c63
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 84ad05fbb8529856645344a81b01a0bbff8818493535de08a5e85b2054adc31a
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .  --verbose
   skip: true  # [py<38]
+  script: 
+    - {{ PYTHON }} -m pip install --no-deps --ignore-installed .  -vv
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - llvm-openmp  # [osx]
+    - llvm-openmp    # [osx]
   host:
     - python
-    - cython >=0.29.24
-    - joblib >=1.0.0
-    - llvm-openmp  # [osx]
-    - numpy   1.19  # [py==38 or py==39]
-    - numpy   1.21  # [py==310]
+    - cython
+    - joblib
```

Comment:
I think joblib is not required in host:
https://github.com/scikit-learn/scikit-learn/blob/1.2.0/sklearn/_min_dependencies.py#L30

Reply:
Thanks. I've removed it